### PR TITLE
docs(blog): add campaign finance blog post

### PR DESCRIPTION
**Pull request by NickCrews**
@cpcloud Here is a draft of my blog post!

I have bad timing, and I'm about to go out of office next week, so I won't be very responsive until Monday March 20. Sorry if this slows down the release cycle you were hoping for!

The plots don't look like they are rendering on the preview. Any idea on how to get that to work?

They look like this to me:
image

### CODE REVIEW

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    ibis.options.repr.interactive.max_columns = None
</pre><p>In 5.0 this won't be necessary I believe. We set the limit to something arbitrarily large (1,000,000 IIRC) when inside a Jupyter notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    kept = conn.create_table("kept", kept)
</pre><p>Since you're using an in-memory connection, <code>.cache()</code> would be a good choice here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #15.    comms = conn.create_table("committees", read_committees())
</pre><p>Could use <code>cache()</code> here too if you want. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    together = kept.left_join(comms, "CMTE_ID").drop("CMTE_ID_x", "CMTE_ID_y")
</pre><p>Could use a selector here if you'd like: <code>.drop(s.startswith("CMTE_ID_"))</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.        return yyyymmdd.cast("Date")
</pre><p>Can you change this to <code>"date"</code>? Technically the parser is case insensitive, but conventionally we've preferred lower case.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.        return first_letter.substitute(election_types, else_=ibis.null())
</pre><p><code>ibis.NA</code> is also fine here </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>We do have <code>bucket</code> and <code>histogram</code> methods, but I suspect they aren't working for you. We should definitely see if we can make them work for your use cases!  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.            median_amount=_.TRANSACTION_AMT.approx_median(),
</pre><p>This might be a good place to use the <code>across</code> selector:</p><p> </p><pre class="ql-syntax" spellcheck="false">table.relabel({"TRANSACTION_AMT": "amount"}).group_by(by).agg(
    n_donations=_.count(),
    s.across(
        s.c("TRANSACTION_AMT"),
        dict(total=_.sum(), mean=_.mean(), median=_.approx_median())
    )
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>Nice comparison!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:56]: Gb on this line should be GB

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:1044]: I would remove this 'First' since you said first just above this line too. Maybe saying 'It's usefull to ...' instead :)

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:1297]: Should this be GB?

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:1359]: This is really interesting!!! Great point 😄

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:2122]: Not sure if it's just me, but the sorted table isn't rendering.

File: docs/blog/rendered/campaign-finance.ipynb
Comment (marlenezw) [inline docs/blog/rendered/campaign-finance.ipynb:2150]: Awesome post Nick!

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>hmmm, I had found <code>histogram()</code>,and indeed it wasn't working. But I hadn't found <code>bucket()</code>, let me try that    </p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Done</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Done.</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>I think I actually prefer being explicit here.</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Oh yeah, this didn't exist when I wrote this.</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>But for this blog post, I'll just remove this to keep it simple.</p>

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:56]: Yeah, better keep it consistent

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1044]: lol, this was me adding/reordering notebook cells willy nilly so all the joiner/ordering words now make no sense. Good catch!

**Comment by NickCrews** [comment]
Thanks for the reviews @cpcloud @marlenezw! I addressed all of your feedback, mostly accepting it.

Still todos:

1. Now I am blocked by a bug. This is preventing me from addressing the other todos. To reproduce with this notebook (macOS 12.6.3, x86):
  a. Create a new, fresh virtual env
  b. `pip install ibis-framework[duckdb]==5.0.0` (eg I'm using the published version, not main)
  c. Run the notebook. You get a kernel crash very early on, when I run `kept.count()`
2. Figure out how to persist the altair charts so they are visible on the docs website. I don't really understand how altair works, but I think [from reading this](https://altair-viz.github.io/user_guide/display_frontends.html#altair-s-renderer-framework) I might try `alt.renderers.enable("png")` as a first attempt. Is the limitation that we need a javascript engine on the docs with a live internet connection, and that isn't available on github pages? This is what the raw output looks like, it looks liek it is trying to fetch from a cdn:
```json
 {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/nickcrews/Documents/projects/scg/ibis/.venv/lib/python3.10/site-packages/altair/utils/core.py:317: FutureWarning: iteritems is deprecated and will be removed in a future version. Use .items instead.\n",
      "  for col_name, dtype in df.dtypes.iteritems():\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<div id=\"altair-viz-4775987a74774988a7b4938711fd55e9\"></div>\n",
       "<script type=\"text/javascript\">\n",
       "  var VEGA_DEBUG = (typeof VEGA_DEBUG == \"undefined\") ? {} : VEGA_DEBUG;\n",
       "  (function(spec, embedOpt){\n",
       "    let outputDiv = document.currentScript.previousElementSibling;\n",
       "    if (outputDiv.id !== \"altair-viz-4775987a74774988a7b4938711fd55e9\") {\n",
       "      outputDiv = document.getElementById(\"altair-viz-4775987a74774988a7b4938711fd55e9\");\n",
       "    }\n",
       "    const paths = {\n",
       "      \"vega\": \"https://cdn.jsdelivr.net/npm//vega@5?noext\",\n",
       "      \"vega-lib\": \"https://cdn.jsdelivr.net/npm//vega-lib?noext\",\n",
       "      \"vega-lite\": \"https://cdn.jsdelivr.net/npm//vega-lite@4.17.0?noext\",\n",
       "      \"vega-embed\": \"https://cdn.jsdelivr.net/npm//vega-embed@6?noext\",\n",
       "    };\n",
       "\n",
       "    function maybeLoadScript(lib, version) {\n",
       "      var key = `${lib.replace(\"-\", \"\")}_version`;\n",
       "      return (VEGA_DEBUG[key] == version) ?\n",
       "        Promise.resolve(paths[lib]) :\n",
       "        new Promise(function(resolve, reject) {\n",
       "          var s = document.createElement('script');\n",
       "          document.getElementsByTagName(\"head\")[0].appendChild(s);\n",
       "          s.async = true;\n",
       "          s.onload = () => {\n",
       "            VEGA_DEBUG[key] = version;\n",
       "            return resolve(paths[lib]);\n",
       "          };\n",
       "          s.onerror = () => reject(`Error loading script: ${paths[lib]}`);\n",
       "          s.src = paths[lib];\n",
       "        });\n",
       "    }\n",
       "\n",
       "    function showError(err) {\n",
       "      outputDiv.innerHTML = `<div class=\"error\" style=\"color:red;\">${err}</div>`;\n",
       "      throw err;\n",
       "    }\n",
       "\n",
       "    function displayChart(vegaEmbed) {\n",
       "      vegaEmbed(outputDiv, spec, embedOpt)\n",
       "        .catch(err => showError(`Javascript Error: ${err.message}<br>This usually means there's a typo in your chart specification. See the javascript console for the full traceback.`));\n",
       "    }\n",
       "\n",
       "    if(typeof define === \"function\" && define.amd) {\n",
       "      requirejs.config({paths});\n",
       "      require([\"vega-embed\"], displayChart, err => showError(`Error loading script: ${err.message}`));\n",
       "    } else {\n",
       "      maybeLoadScript(\"vega\", \"5\")\n",
       "        .then(() => maybeLoadScript(\"vega-lite\", \"4.17.0\"))\n",
       "        .then(() => maybeLoadScript(\"vega-embed\", \"6\"))\n",
       "        .catch(showError)\n",
       "        .then(() => displayChart(vegaEmbed));\n",
       "    }\n",
       "  })({\"config\": {\"view\": {\"continuousWidth\": 400, \"continuousHeight\": 300}}, \"data\": {\"name\": \"data-b19b5f45853c1a9ea92ff443230ae01b\"}, \"mark\": \"bar\", \"encoding\": {\"x\": {\"field\": \"month_str\", \"sort\": [\"Jan\", \"Feb\", \"Mar\", \"Apr\", \"May\", \"Jun\", \"Jul\", \"Aug\", \"Sep\", \"Oct\", \"Nov\", \"Dec\"], \"type\": \"ordinal\"}, \"y\": {\"field\": \"n_donations\", \"type\": \"quantitative\"}}, \"$schema\": \"https://vega.github.io/schema/vega-lite/v4.17.0.json\", \"datasets\": {\"data-b19b5f45853c1a9ea92ff443230ae01b\": [{\"month_int\": null, \"n_donations\": 1514, \"total_amount\": 250297, \"mean_amount\": 165.3216644649934, \"median_amount\": 100, \"month_str\": null}, {\"month_int\": 1.0, \"n_donations\": 348979, \"total_amount\": 174837854, \"mean_amount\": 500.9982090612902, \"median_amount\": 121, \"month_str\": \"Jan\"}, {\"month_int\": 2.0, \"n_donations\": 581646, \"total_amount\": 255997655, \"mean_amount\": 440.126219384299, \"median_amount\": 100, \"month_str\": \"Feb\"}, {\"month_int\": 3.0, \"n_donations\": 1042577, \"total_amount\": 430906797, \"mean_amount\": 413.3093258339672, \"median_amount\": 81, \"month_str\": \"Mar\"}, {\"month_int\": 4.0, \"n_donations\": 1088244, \"total_amount\": 299252692, \"mean_amount\": 274.98676032213365, \"median_amount\": 50, \"month_str\": \"Apr\"}, {\"month_int\": 5.0, \"n_donations\": 1374248, \"total_amount\": 387317202, \"mean_amount\": 281.83937833637015, \"median_amount\": 48, \"month_str\": \"May\"}, {\"month_int\": 6.0, \"n_donations\": 1667285, \"total_amount\": 465305247, \"mean_amount\": 279.07960966481437, \"median_amount\": 44, \"month_str\": \"Jun\"}, {\"month_int\": 7.0, \"n_donations\": 1607053, \"total_amount\": 320528605, \"mean_amount\": 199.45117242555162, \"median_amount\": 35, \"month_str\": \"Jul\"}, {\"month_int\": 8.0, \"n_donations\": 2023466, \"total_amount\": 473544182, \"mean_amount\": 234.02626088108227, \"median_amount\": 35, \"month_str\": \"Aug\"}, {\"month_int\": 9.0, \"n_donations\": 2583847, \"total_amount\": 697888624, \"mean_amount\": 270.0967294116099, \"median_amount\": 38, \"month_str\": \"Sep\"}, {\"month_int\": 10.0, \"n_donations\": 3686024, \"total_amount\": 850820707, \"mean_amount\": 230.82343115508743, \"median_amount\": 29, \"month_str\": \"Oct\"}, {\"month_int\": 11.0, \"n_donations\": 2545616, \"total_amount\": 285143995, \"mean_amount\": 112.01375030640914, \"median_amount\": 25, \"month_str\": \"Nov\"}, {\"month_int\": 12.0, \"n_donations\": 2119311, \"total_amount\": 283081648, \"mean_amount\": 133.57249030463203, \"median_amount\": 25, \"month_str\": \"Dec\"}]}}, {\"mode\": \"vega-lite\"});\n",
       "</script>"
      ],
      "text/plain": [
       "alt.Chart(...)"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "months_in_order = list(month_map.values())\n",
    "alt.Chart(by_month.execute()).mark_bar().encode(\n",
    "    x=alt.X('month_str:O', sort=months_in_order),\n",
    "    y='n_donations:Q',\n",
    ")"
   ]
  },
```
3. try using the built in `bucket()` instead of my custom version.

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [reply docs/blog/rendered/campaign-finance.ipynb:1]: SGTM!

Comment (cpcloud) [reply docs/blog/rendered/campaign-finance.ipynb:1]: SGTM, let's keep it as is then!

**Comment by cpcloud** [comment]
I don't think GitHub runs notebook code with arbitrary JavaScript, so you'll need a way to render the plots statically as a PNG, which is what is happening in [this notebook](https://ibis-project.org/blog/rendered/ci-analysis/).

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:2128]: See my longer response below, still figuring this out.

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>Would you mind if I put a gzipped version of the CSV into our <code>ibis-tutorial-data</code> GCS bucket to avoid the download and the extra code it requires? This also amplifies the effect of caching :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.    raw = conn.read_csv(csv_path)
</pre><p>How about using <code>ibis.read_csv</code> instead of an explicit connection?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.        return yyyymmdd.cast("date")
</pre><p>This entire function can be turned into a one-line:</p><p> </p><pre class="ql-syntax" spellcheck="false">return val.cast(str).lpad(8, "0").to_timestamp("%m%d%Y").date()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>I think I actually slightly lean towards keeping this as is. I think the current method is more realistic and is easier for the reader to empathize with. With the GCS it would feel more like a cute example than a real task. The current method isn't <strong>that</strong> long. If you feel strongly though then I can update.</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>uggghhhhh dang I looked for this, and I think I found to_timestamp() but then ignored it because it returned a TimeStamp, I didn't make the connection that then I merely had to call .date() to extract the Date.</p><p> </p><p>Probably this is just me, but perhaps this is a documentation issue? Or perhaps DateValue and TimeStampValue should .parse_string() classmethods? When I went to the docs I went to these datatypes at the start of my search but I didn't find anything there.</p>

Comment (cpcloud) [reply docs/blog/rendered/campaign-finance.ipynb:1]: I think it's a documentation issue. We need a cookbook of sorts, or at least some categorical (e.g., Timestamp Operations) how-to content.

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Great! I thought I needed it so that the contributions and committees tables would be in the same DB, but I see that they still are with ibis.read_csv()</p>

**Comment by NickCrews** [comment]
To get .png outputs, for posterity (posting this here because I don't want to pollute the notebook with this side quest)

1. `pip install altair_saver`
2. install the deps for altair_saver: `pip install selenium`
3. Install chromedriver: `pip install chromedriver-binary-auto`. Then, run one time, to download the chromedriver binary and add it to PATH ([from here](https://pypi.org/project/chromedriver-binary-auto/)): `import chromedriver_binary`
4. `alt.renderers.enable("png")` should work!

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Nope, we can leave it as is!</p>

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>It looks you may have <code>ipywidgets</code> installed, which apparently allows the progress bar to work. Any chance you can run the notebook without that so there's <em>only</em> the number showing up in the output?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>Should this be uncommented?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>These plots great!</p><p> </p><p>Any chance we can make them a bit bigger? They feel kind of cramped.</p><p> </p><p>If it's a pain, don't worry about it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #5.    import chromedriver_binary
</pre><p>Wow, this is sort of terrifying, but eh on the other hand ... ship it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (cpcloud) [reply docs/blog/rendered/campaign-finance.ipynb:1]: `ipywidgets` requires JavaScript to properly show up, so I'd rather leave it out if we can.

**Comment by cpcloud** [comment]
@NickCrews I don't think we should do it now, but given the rigmarole you've gone through to get PNGs working I **highly** recommend [`plotnine`](https://plotnine.readthedocs.io/en/stable/index.html) for declarative plotting. It's a bit to wrap your head around at first, but it pays off IMO.

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: Yeah, I got around the bug by installing ipyidgets. But I think you're right. I should just update ibis to main and then rerun this without the progress bar. Will do!

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>oops, yes.</p>

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>.bucket() works, I migrated to that.</p>

File: docs/blog/rendered/campaign-finance.ipynb
Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Done in v4</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>I changed to svg and ran <code>alt.renderers.set_embed_options(width=500, height=600)</code>,which I think looks a little better.  </p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>Yeah, some black magic going on there. Don't look too closely ;)</p>

**Comment by NickCrews** [comment]
OK, in v4 I
- removed the progress bars
- made the charts bigger
- fixed the mistakenly-commented-out code

re: plotnine, I agree that seems better suited for this case. Altair is pretty sweet though when you need interaction or want to post-tweak the output to make a UI, I recommend checking it out if you haven't.

File: docs/blog/rendered/campaign-finance.ipynb
Comment (cpcloud) [inline docs/blog/rendered/campaign-finance.ipynb:1]: <p>Looks like the plots aren't showing up for whatever reason :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: <p>oops I was playing around with the renderers and I accidentally have it swapped to .svg. Fixing now.</p>

Comment (NickCrews) [reply docs/blog/rendered/campaign-finance.ipynb:1]: v5 pushed

**Review by cpcloud — APPROVED** [review]
SHIP IT❗

@NickCrews Can you squash to a single `docs(blog): ...` commit? Will merge on green after that!

**Comment by NickCrews** [comment]
What a journey! Thanks for all the hard work on editing this! Glad to be done ;)

**Comment by cpcloud** [comment]
Excellent work! Merging!!

**Pull request merged**
